std/conv.d

```diff
@@ -207,6 +207,13 @@ template to(T)
     {
         return toImpl!T(arg);
     }
+
+    // Fix issue 16108
+    T to(S)(ref S arg)
+        if (isAggregateType!S && !isCopyable!S)
+    {
+        return toImpl!T(arg);
+    }
 }
```

Comment:
I don't follow. Right now we have overloads in `to` **and** overloads in `toImpl`, with "fix issue" `to` overloads having the same constraints as their `toImpl` counterparts. Can we not have just the `toImpl` overloads? What am I missing?